File: Language-C/src/for2.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    system("clear");
    
    int ano, contador = 0;

    for(ano = 1950; ano <= 2026; ano++) {
        if(ano % 4 == 0) {
            printf("O ano %d é bissexto\n", ano);
            contador++;
        }
    }

    printf("\nExiste %d anos bissextos entre 1950 e 2026", contador);

    return 0;
}

File: UC9/Language-Py/src/for2.py
contador = 0
for ano in range(1950, 2027):
    if (ano % 4 == 0 and ano % 100 != 0) or (ano % 400 == 0):
        print(f"{ano} é um ano bissexto.")
        contador += 1

print(f"Total de anos bissextos entre 1950 e 2026: {contador}")